doc/source/reference/array_api.rst

```diff
@@ -0,0 +1,787 @@
+   * - ``matrix_rank`` does not support 1-dimensional arrays.
+     - **Breaking**
+     -
+   * - New function ``matrix_transpose``.
+     - **Compatible**
+     - Unlike ``np.transpose``, ``matrix_transpose`` only transposes the last
+       two axes. See `the spec definition
+       <https://data-apis.org/array-api/latest/API_specification/generated/signatures.linear_algebra_functions.matrix_transpose.html#signatures.linear_algebra_functions.matrix_transpose>`__
+   * - ``outer`` only supports 1-dimensional arrays.
+     - **Breaking**
+     -
+   * - ``pinv`` has an ``rtol`` keyword argument instead of ``rcond``
+     - **Compatible**
+     - The meaning of ``rtol`` and ``rcond`` is the same, but the default
+       value for ``rtol`` is ``max(M, N) * eps``, whereas the default value
+       for ``rcond`` is ``1e-15``.
+   * - ``solve`` only accepts ``x2`` as a vector when it is exactly
+       1-dimensional.
+     - **Breaking**
+     - The ``np.linalg.solve`` behavior is ambiguous. See `this numpy issue
+       <https://github.com/numpy/numpy/issues/15349>`__ and `this array API
+       specification issue
+       <https://github.com/data-apis/array-api/issues/285>`__ for more
+       details.
+   * - New function ``svdvals``.
+     - **Compatible**
+     - Equivalent to ``np.linalg.svd(compute_uv=False)``.
+   * - The ``axis`` keyword to ``tensordot`` must be a tuple.
+     - **Compatible**
+     - In ``np.tensordot``, it can also be an array or array-like.
+   * - ``trace`` operates on the last two axes.
+     - **Breaking**
+     - ``np.trace`` operates on the first two axes by default. Note that the
+       array API ``trace`` does not allow specifying which axes to operate on.
+
+Manipulation Functions Differences
+----------------------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
+     - Type
+     - Notes
+   * - Various functions have been renamed
+     - **Compatible**
+     - See :ref:`array_api-name-changes`.
+   * - ``concat`` has different default casting rules from ``np.concatenate``
+     - **Strictness**
+     - No cross-kind casting. No value-based casting on scalars.
+   * - ``stack`` has different default casting rules from ``np.stack``
+     - **Strictness**
+     - No cross-kind casting. No value-based casting on scalars.
+   * - New function ``permute_dims``.
+     - **Compatible**
+     - Unlike ``np.transpose``, the ``axis`` keyword argument to
+       ``permute_dims`` is required.
+
+Set Functions Differences
+-------------------------
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
+     - Type
+     - Notes
+   * - New functions ``unique_all``, ``unique_counts``, ``unique_inverse``,
+       and ``unique_values``.
+     - **Compatible**
+     - See :ref:`array_api-name-changes`.
+   * - The four ``unique_*`` functions return a named tuple.
+     - **Compatible**
+     -
+   * - ``unique_all`` and ``unique_indices`` return indices with the same
+       shape as ``x``.
```

Comment:
Given that these are new function names, the `breaking` label seems wrong - this is `compatible`. It's good to point out separately as a note so fine to keep, but it's backwards compatible.

Reply:
Ah that's a good point. I made that same observation for some other new functions, but missed it here.